custom_components/frigate/media_source.py

```diff
@@ -710,10 +730,12 @@ async def _get_event_summary_data(
         """Get event summary data."""
 
         try:
+            info = await system_info.async_get_system_info(self.hass)
+
             if identifier.frigate_media_type == FrigateMediaType.CLIPS:
-                kwargs = {"has_clip": True}
+                kwargs = {"has_clip": True, "timezone": info.get("timezone", "utc")}
             else:
-                kwargs = {"has_snapshot": True}
+                kwargs = {"has_snapshot": True, "timezone": info.get("timezone", "utc")}
```

Comment:
Can avoid the double setting of the timezone, by just moving the timezone argument down into the call to `async_get_event_summary` next to `kwargs`.